lib/block_view_test.go

```diff
@@ -489,10 +498,17 @@ func TestBasicTransfer(t *testing.T) {
 	_ = assert
 	_ = require
 
-	chain, params, db := NewLowDifficultyBlockchain()
+	postgres := InitializeTestPostgresInstance(t)
```

Comment:
IMO this logic should be refactored into some test_utils. Individual tests shouldn't have to know the specifics of a postgres instance initialization. Also ideally we would have one test and not two sister tests like `_testBasicTransfer` and `_testBasicTransferWithPostgres`.